src/data/nat/basic.lean

```diff
@@ -913,6 +919,13 @@ begin
   { intros h, rw h },
 end
 
+@[simp]
+lemma div_by_eq_iff_eq_of_dvd_dvd {a b d : ℕ} (hda : d ∣ a) (hdb : d ∣ b) : a / d = b / d ↔ a = b :=
```

Comment:
I don't thinks this use of `by` matches our naming convention. Anyway, to match [`div_left_inj`](https://leanprover-community.github.io/mathlib_docs/algebra/group/basic.html#div_left_inj) I think this should be:

```suggestion
protected lemma div_left_inj {a b d : ℕ} (hda : d ∣ a) (hdb : d ∣ b) : a / d = b / d ↔ a = b :=
```